app/src/main/java/org/simple/clinic/home/overdue/OverdueUpdate.kt

```diff
@@ -25,6 +26,71 @@ class OverdueUpdate(
       is ShareOverdueListClicked -> shareOverdueListClicked(event)
       is OverdueAppointmentsLoaded -> overdueAppointmentsLoaded(event, model)
       PendingListFooterClicked -> pendingListFooterClicked(model)
+      is ChevronClicked -> chevronClicked(model, event.headerText)
+    }
+  }
+
+  private fun chevronClicked(model: OverdueModel, headerText: Int): Next<OverdueModel, OverdueEffect> {
+    val updatedModel = when (headerText) {
+      R.string.overdue_pending_to_call_header -> {
+        pendingChevronStateIsChanged(model)
```

Comment:
As Sasi mentioned in his [message](https://obvious-hq.slack.com/archives/G018FHX0A5R/p1654748647803329).. We should use enums or something instead of the string res for identification.